### PR TITLE
Angular routing

### PR DESCRIPTION
**Pull request by xoddong**

### CODE REVIEW

**Comment by xoddong** [comment]
:100:

File: app/views/layout.server.view.html [hunk 1]
```diff
@@ -35,89 +35,16 @@
...
-		}
-
-		#over_map { 
-			position: absolute;
-			top: 10px; 
-			left: 10px; 
-			z-index: 99; 
-		}
 
+	<style type="text/css">
```
Comment (tesk9) [inline app/views/layout.server.view.html]: If there's no styling here, we should just delete the style tag

File: app/views/layout.server.view.html [hunk 2]
```diff
@@ -35,89 +35,16 @@
...
-			left: 10px; 
-			z-index: 99; 
-		}
 
+	<style type="text/css">    
   </style>
 </head>
 
 <body class="ng-cloak">
 	<header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top navbar-inverse"></header>
```
Comment (tesk9) [inline app/views/layout.server.view.html]: formatting

File: app/views/layout.server.view.html [hunk 3]
```diff
@@ -35,89 +35,16 @@
...
-
-  <section class="content">
-    <section class="container">
-      {% block content %}{% endblock %}
-    </section>
-  </section>
-
+	<section>
+		{% block content %}{% endblock %}
```
Comment (tesk9) [inline app/views/layout.server.view.html]: still formatting. we're doing 2 spaces for indentation, right?

File: app/views/layout.server.view.html [hunk 4]
```diff
@@ -129,96 +56,14 @@
...
-		<!-- <input  type="text" class="form-control" placeholder="checking if this works"> -->
-  	<div id="map-canvas" class="col-md-9"></div>
 
-		<div id="pac-back" class="col-md-3 search-form">
-			<input id="pac-input" class="controls form-control" type="text" placeholder="Search Box">
-		</div>	
-  
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
   <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
 	<script type="text/javascript" src="http://{{request.hostname}}:35729/livereload.js"></script>
```
Comment (tesk9) [inline app/views/layout.server.view.html]: formatting

File: public/config.js [hunk 5]
```diff
@@ -4,7 +4,14 @@
 var ApplicationConfiguration = (function() {
 	// Init module configuration options
 	var applicationModuleName = 'concrete';
-	var applicationModuleVendorDependencies = ['ngResource', 'ngCookies',  'ngAnimate',  'ngTouch',  'ngSanitize',  'ui.router', 'ui.bootstrap', 'ui.utils'];
+	var applicationModuleVendorDependencies = ['ngResource',
```
Comment (tesk9) [inline public/config.js]: why was this change made? From listing dependencies on one line to shoving them all the way over?

File: public/config.js [hunk 6]
```diff
@@ -4,7 +4,14 @@
...
+																						 'ngCookies',
+																						 'ngAnimate',
```
Comment (tesk9) [inline public/config.js]: why the white space?

File: public/modules/core/controllers/home.client.controller.js [hunk 7]
```diff
@@ -5,5 +5,11 @@ angular.module('core').controller('HomeController', ['$scope', 'Authentication',
 	function($scope, Authentication) {
 		// This provides Authentication context.
 		$scope.authentication = Authentication;
-	}
+
+    $scope.map = { center: { latitude: 45, longitude: -73 }, zoom: 8 };
+    var events = {
+      places_changed: function (searchBox) {}
+    }
```
Comment (tesk9) [inline public/modules/core/controllers/home.client.controller.js]: semi-colon

File: public/modules/core/controllers/home.client.controller.js [hunk 8]
```diff
@@ -5,5 +5,11 @@ angular.module('core').controller('HomeController', ['$scope', 'Authentication',
 	function($scope, Authentication) {
```
Comment (tesk9) [inline public/modules/core/controllers/home.client.controller.js]: inconsistent indentation here

File: public/modules/core/css/core.css [hunk 9]
```diff
@@ -1,15 +1,69 @@
+.angular-google-map-container {
+  height: 100vh;
+  width: 100%;
+}
+
+.google-maps {
+  padding: 0;
+}
+
+/*.content {*/
```
Comment (tesk9) [inline public/modules/core/css/core.css]: is this going to be taken out? let's either erase or uncomment

File: public/modules/core/css/core.css [hunk 10]
```diff
@@ -1,15 +1,69 @@
...
+/*}*/
+
+html, body, #map-canvas { 
+  height: 96vh; 
+  margin-top: 3vh; 
+  padding-top: 3 vh;
+}
+
 .content {
 	margin-top: 50px;
```
Comment (tesk9) [inline public/modules/core/css/core.css]: too much indentation

**Pull request merged**